Assets/HoloToolkit-Preview/QRTracker/Scripts/QRCodesManager.cs

```diff
@@ -0,0 +1,160 @@
+﻿using System;
+using System.Collections;
+
+using System.Collections.Generic;
+
+using UnityEngine;
+
+using QRCodesTrackerPlugin;
+namespace HoloToolkit.Unity.QRTracking
+{
+    public static class QRCodeEventArgs
+    {
+        public static QRCodeEventArgs<TData> Create<TData>(TData data)
```

Comment:
Why do we have an extra layer instead of just a new directly?

Reply:
It is templated used for events.